### PR TITLE
[depends] use pre-compiled nasm instead of building it.

### PR DESCRIPTION
**Pull request by afedchin**
this also speed up built on windows platforms.

### CODE REVIEW

File: depends/windows/nasmw/nasmw.txt [hunk 1]
```diff
@@ -1 +1 @@
-nasmw http://mirrors.xbmc.org/build-deps/sources/nasm-2.11.06.zip
+nasmw http://www.nasm.us/pub/nasm/releasebuilds/2.12.02/win32/nasm-2.12.02-win32.zip
```
Comment (notspiff) [inline depends/windows/nasmw/nasmw.txt]: shouldn't we host ourself?

Comment (afedchin) [reply depends/windows/nasmw/nasmw.txt]: what for? we are using dependencies already which are hosted on 3rd party servers.

Comment (notspiff) [reply depends/windows/nasmw/nasmw.txt]: okidoki. earlier we wanted to host everything ourself to ensure reproducibility.

Comment (MartijnKaijser) [reply depends/windows/nasmw/nasmw.txt]: we always put them on our servers in the past cause other weren't that reliable

**Comment by afedchin** [comment]
any news on this?

File: depends/windows/nasmw/nasmw.txt (hunk 1 above)
Comment (MartijnKaijser) [reply depends/windows/nasmw/nasmw.txt]: i just uploaded the file under win32 on the mirrors

Comment (afedchin) [reply depends/windows/nasmw/nasmw.txt]: the URL was updated.

Comment (FernetMenta) [reply depends/windows/nasmw/nasmw.txt]: why don't we trash this addon. imo this is rather useless because ffmpeg can encode flac too.

**Comment by Rechi** [comment]
compiling from source is preferred

**Pull request closed without merging**